data.sql

```diff
@@ -11,3 +11,19 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon','June 7, 2005',7,'1',20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom','October 13, 1998',3,'1',17);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto','May 14, 2022',4,'1',22); 
+
```

Comment:
### Optional
- [ ] It would be better to include the Smith Adam data as it is required. For example; 
```
INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);
```
However, this will not prevent us from approving your project